hw/ip/otbn/data/otbn.hjson

```diff
@@ -134,17 +134,34 @@
   registers: [
     { name: "CMD"
       desc: "command register",
-      swaccess: "r0w1c",
+      swaccess: "wo",
       hwaccess: "hro",
       hwext: "true",
       hwqe:  "true",
       fields: [
-        { bits: "0"
-          name: "start"
+        { bits: "1:0"
+          name: "cmd"
+          resval: 0,
           desc: '''
-            Start the operation
+            Initiates an OTBN operation.
+
+            1 (EXECUTE): Start the execution of the program stored in the
+               instruction memory, starting at address !!START_ADDR.
+
+            2 (SEC_WIPE_DMEM): Securely remove all contents from the data
+              memory.
 
-            The completion is signalled by the done interrupt.
+            3 (SEC_WIPE_IMEM): Securely remove all contents from the instruction
+              memory.
+
+            During the operation the command is reflected in the !!STATUS
+            register, indicating that OTBN is busy performing the requested
+            operation.
+
+            All commands signal their completion by raising the done interrupt;
+            alternatively, software may poll the !!STATUS register.
+
+            Unrecognized commands are ignored.
```

Comment:
Do we specify what happens if you write this register when OTBN is busy? Maybe we should say it here too?

Reply:
Good point, I don't think we specify that at the moment.

I see two options:
- Ignore writes while OTBN is busy.
- Issue a fatal error while OTBN is busy. This matches what we do for IMEM/DMEM writes while OTBN is busy.

I went for the "ignore" part so far to break a secure wipe chain: if you're running a secure wipe command, the fatal error would also cause a secure wipe. So you have a situation where you have issued a secure wipe command of, say, the IMEM, then write to CMD while OTBN is busy, which causes a fatal error, which then causes a secure wipe of all state, and then you still need to complete the command by sending a done interrupt. 

That's possible to implement and also not much of an exception to our logic. But I'm not convinced we'd gain anything over just ignoring the write.

Reply:
I don't have strong feelings on what the behaviour should be (except that we should specify it! :-)), but don't we already have that chain if there's a bus access to memory while we're doing the wipe?

Reply:
I went for the "ignore invalid commands" approach and documented it.